examples/orientation_parser_planar.html

```diff
@@ -0,0 +1,116 @@
+            new itowns.PlanarControls(view, {
+                // We do not want the user to zoom out too much
+               maxAltitude: 40000000,
+                // We want to keep the rotation disabled, to only have a view from the top
+                enableRotation: true,
+                // Faster zoom in/out speed
+               zoomInFactor: 0.5,
+               zoomOutFactor: 0.5,
+                // Don't zoom too much on smart zoom
+               smartZoomHeightMax: 100000,
+            });
+            
+            // Input data : a geoJson file, with point features, with orientation specific properties
+            var panoramics = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            var layer = {
+                crsOut: view.referenceCrs,
+            };
+            // Input data are given in Lambert93 projection
+            itowns.proj4.defs('EPSG:2154',
+                '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+            
+            // Add a promise to load oriented point datas
+            var promises = [];
+            promises.push(
+                // first parse the JSON format
+                itowns.Fetcher.json(panoramics, {})
+                // then get geographic informations (coordinates) using GeoJson parser
+                .then(orientations => itowns.GeoJsonParser.parse(orientations, layer))
+                // then get orientation informations 
+                .then(collection => itowns.OrientationParser.parse(collection.features, layer))
+            );        
+            
+            // load collada model of a renault trafic
+            promises.push(ThreeLoader.load('Collada',
+             'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/renault_trafic_no_texture.dae'));
+
+            // wait for all the promises
+            Promise.all(promises).then(res => loadFeatures(res[0], res[1]));
+            
+            // place a 3d model oriented for each feature
+            function loadFeatures(features, collada) {
+
+                // prepare collada model base scale and rotation. in this example, we want the car to go straigth.
+                var colladaModel = collada.scene;
+                colladaModel.scale.set(0.5, 0.5, 0.5);
+                colladaModel.rotateZ(-Math.PI / 2 );
+
+                for (const feature of features) {                  
+                
+                    // clone the collada model
+                    var model = colladaModel.clone();
+                    
+                    // get point coordinate
+                    var coord = feature.vertices[0]; 
+                    // set the ground altitude
+                    coord = coord.as('EPSG:4326');
+                    coord.setAltitude(0);
```

Comment:
I think you can remove these 3 lines : `coord` is unused

Reply:
yes, thanks